src/main/scala/ch/openolitor/buchhaltung/BuchhaltungRoutes.scala

```diff
@@ -442,7 +442,7 @@ trait BuchhaltungRoutes extends HttpService with ActorReferences
       case (Some(iban), Some(creditorIdentifier)) => {
         val emptyIbanList = checkEmptyIban(rechnungen)
         if (emptyIbanList.isEmpty) {
-          val xmlText = Pain008_001_07_Export.exportPain008_001_07(rechnungen, kontoDatenProjekt, NbOfTxs, projekt)
```

Comment:
Pain008_001_07_Export is not used any more? Do we delete it or leave for (possibly) later?

Reply:
i would like to leave it as this is the latest version and it is possibly used in other countries. The version I have introduced here is the version that was agreed by german banks to be used. In case we need to enable it in the future it would be easier to leave it. What do you think??

Reply:
ok